apps/forklift/lib/forklift/message_handler.ex

```diff
@@ -21,11 +21,14 @@ defmodule Forklift.MessageHandler do
   Handle each kafka message.
   """
   def handle_messages(messages, %{dataset: %SmartCity.Dataset{} = dataset}) do
-    messages
-    |> Enum.map(&parse/1)
-    |> Enum.map(&yeet_error/1)
-    |> Enum.reject(&error_tuple?/1)
-    |> Forklift.DataWriter.write(dataset: dataset)
+    timed_messages =
+      messages
+      |> Enum.map(&parse/1)
+      |> Enum.map(&yeet_error/1)
+      |> Enum.reject(&error_tuple?/1)
+      |> Forklift.DataWriter.write(dataset: dataset)
+
+    Task.start(fn -> Forklift.DataWriter.write_to_topic(timed_messages) end)
```

Comment:
Is there anything we should be doing once this task is complete? Even just debug logging the end result?

Reply:
We're putting a debugge message in there